templates/Gemfile.erb

```diff
@@ -13,7 +14,6 @@ gem "jquery-rails"
 gem "newrelic_rpm", ">= 3.9.8"
 gem "pg"
 gem "rack-canonical-host"
-gem "rails", "<%= Suspenders::RAILS_VERSION %>"
 gem "recipient_interceptor"
 gem "sass-rails", "~> 5.0"
 gem "simple_form"
```

Comment:
Why move `rails` to top? When the frameworks are injected into this file, do they get added immediately after line 4 or are they added to the "paragraph" of gems below?